docs: update the quickstart for Next.js (#4221)
neondatabase/website: content/docs/auth/quick-start/nextjs.md

  </RightCode>
</TwoColumnStep>

<TwoColumnStep title="Add Neon Auth styles">
  <LeftContent>

Import the Neon Auth UI styles in your `app/globals.css` file. Add this line at the top of the file:

  </LeftContent>
  <RightCode label="app/globals.css">

```css
@import "tailwindcss";
@import "@neondatabase/neon-js/ui/css"; // [!code ++]

```

  </RightCode>
</TwoColumnStep>

<Admonition type="note">

> Now that the Auth provider and styles set setup, as a last step lets build some actual pages so you can signup and signin. 

Reviewer: ```suggestion
Now that the Auth provider and styles are set up, let’s build the pages for signing up and signing in
```
